bin/multi-tester

```diff
@@ -43,4 +44,15 @@ if (!getLoader()) {
     exit(1);
 }
 
-exit((new MultiTester())->run($argv) ? 0 : 1);
+try {
+    $code = (new MultiTester())->run($argv) ? 0 : 1;
+} catch (\Exception $e) {
+    if (Summary::isColorSupported()) { // is colored?
+        $errorString = " \033[41;1;97m  ERROR  \033[0m\033[31m {$e->getMessage()} \033[0m" ;
+    } else {
+        $errorString = 'ERROR:' . $e->getMessage();
+    }
+    echo PHP_EOL . $errorString . PHP_EOL;
+    exit(1);
+}
+exit($code);
```

Comment:
What is annoying for you can be useful for other people. Stack trace exists because it can help to debug an error. It should remain visible at least when the `--verbose` option is in use. And if the stack trace is hidden when the output is not verbose, errors should be followed by a tip `Use --verbose option to get more details.`